internal/pkg/create/sample_test.go

```diff
@@ -98,7 +98,7 @@ func TestSamples_GetSampleRepos(t *testing.T) {
 			req, err = http.NewRequest("GET", "https://api.github.com/orgs/slack-samples/repos?type=public", nil)
 			require.NoError(t, err)
 			req.Header.Set("Accept", "application/vnd.github+json")
-			req.Header.Set("X-GitHub-Api-Version", "2022-11-28")
+			req.Header.Set("X-GitHub-API-Version", "2022-11-28")
```

Comment:
```suggestion
			req.Header.Set("X-GitHub-Api-Version", "2022-11-28")
```

🔍 I'm curious if a change to the `sample.go` file without a change here might fail the test?

Reply:
It should fail 🤔 

Commit 81569f1 reverts this change!